planning/motion_velocity_planner/autoware_motion_velocity_obstacle_stop_module/src/obstacle_stop_module.hpp

```diff
@@ -78,6 +88,10 @@ class ObstacleStopModule : public PluginModuleInterface
   mutable std::shared_ptr<DebugData> debug_data_ptr_{};
   std::vector<StopObstacle> prev_closest_stop_obstacles_{};
   std::vector<StopObstacle> prev_stop_obstacles_{};
+
+  // PointCloud-based stop obstacle history
+  std::vector<StopObstacle> stop_pc_obstacle_history_;
```

Comment:
`pc` seems the abbreviation of the pointcloud, but I guess we shouldn't make it short in this case for readability.

Reply:
Other variables uses `prev_`, so `prev_poincloud_obstacles` or something sounds better?